packages/sitecore-jss-rendering-host/src/devServer.ts

```diff
@@ -196,7 +196,22 @@ export function startDevServer({
     serverOptions.public = tunnelUrl;
   }
 
-  const modulePath = path.join(buildArtifactsPath, serverBundleFileName);
+  type ServerBundleFileName =
+    | string
+    | ((pathData: webpack.PathData, assetInfo?: webpack.AssetInfo | undefined) => string);
+
+  const serverBundleFileName: ServerBundleFileName =
+    customServerBundleFileName || configs?.ssrWebpackConfig?.output?.filename || 'server.bundle.js';
+
+  let ServerBundleOutput: string;
```

Comment:
Usually capitalized letter used when you define class, component
```suggestion
  let resolvedServerBundleFilename: string;
```